### PR TITLE
update travis

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

**Comment by juliangruber** [comment]
👍

File: .travis.yml [hunk 1]
```diff
@@ -16,8 +16,8 @@ before_install:
   - export JOBS=max
 
 node_js:
+  - '7'
   - '6'
   - '5'
```
Comment (rvagg) [inline .travis.yml]: remove 5, its gone

**Comment by rvagg** [comment]
We're going `^` for devDeps now? there's an interesting mix in there now. Doesn't bother me if there's a good reason for it.

File: .travis.yml (hunk 1 above)
Comment (ralphtheninja) [inline .travis.yml]: Will do.

**Comment by ralphtheninja** [comment]
> We're going ^ for devDeps now? there's an interesting mix in there now. Doesn't bother me if there's a good reason for it.

I updated to ^ for devDeps just to get rid of updating things like tape etc. I can update all to use ^ if that's alright.

File: .travis.yml (hunk 1 above)
Comment (ralphtheninja) [inline .travis.yml]: And yes, I'll squash later :)

**Comment by ralphtheninja** [comment]
Putting back `.dntrc` with updated node versions. I also want to squash this before merge.

File: .dntrc [hunk 2]
```diff
@@ -0,0 +1,19 @@
+## DNT config file
+## see https://github.com/rvagg/dnt
+
+NODE_VERSIONS="\
+  master   \
+  v7.x-staging  \
+  v6.x-staging  \
+  v4.x-staging  \
```
Comment (ralphtheninja) [inline .dntrc]: @rvagg I picked these branches because I thought it might be interesting to test on running targets. Feel free to give some input here :)

**Pull request merged**